airflow/utils/db.py

```diff
@@ -887,9 +889,9 @@ def _check_migration_errors(session=None) -> Iterable[str]:
         check_run_id_null,
         check_task_tables_without_matching_dagruns,
     ):
-        yield from check_fn(session)
+        yield from check_fn(session)  # type: ignore
```

Comment:
I think it’s better to annotate the list of functions instead.

Reply:
Right ! much better!